### PR TITLE
Simplify Esirkepov formula even more

### PR DESCRIPTION
**Pull request by RemiLehe**
This is a follow-up PR to #2373 .
It makes the formulas more symmetrical and more readable (at least in my opinion). Note that the new formulas are mathematically equivalent to the old ones.

### CODE REVIEW

File: Source/Particles/Deposition/CurrentDeposition.H [hunk 1]
```diff
@@ -398,6 +398,8 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
 #endif
 
     Real const clightsq = 1.0_rt / ( PhysConst::c * PhysConst::c );
+    Real constexpr one_third = 1.0_rt / 3.0_rt;
+    Real constexpr one_sixth = 1.0_rt / 3.0_rt;
```
Comment (RemiLehe) [inline Source/Particles/Deposition/CurrentDeposition.H]:
```suggestion
    Real constexpr one_sixth = 1.0_rt / 3.0_rt;
```
```suggestion
    Real constexpr one_sixth = 1.0_rt / 6.0_rt;
```

**Review by ax3l — APPROVED** [review]
LGTM, thank you! :)

**Review by EZoni — COMMENTED** [review]
Thank you, Remi! Looks good to me, too. The CI test `PEC_particle` fails (see raw output copied below). The relative errors look quite large, but the quantities themselves are zero to machine precision (indeed the absolute errors are also zero to machine precision). Hence, I guess it should be safe to reset the benchmarks for this particular test. I think it might be worth coming up with at least a guess of why this would be the only CI test sensitive to round-off errors due to the new order of arithmetic operations in the current deposition kernel, I don't have a good guess at the moment.

```
working on test: PEC_particle
   re-making clean...
   building...
   found pre-built executable for this test
   copying files to run directory...
   path to input file: Examples/Tests/PEC/inputs_particle_PEC_3d
   running the test...
   mpiexec -n 2 ./main3d.gnu.TEST.TPROF.MTMPI.OMP.QED.OPMD.PSATD.GPUCLOCK.ex inputs_particle_PEC_3d  diag1.file_prefix=PEC_particle_plt   amrex.abort_on_unused_inputs=1 amrex.fpe_trap_invalid=1 amrex.fpe_trap_zero=1 amrex.fpe_trap_overflow=1
   WARNING: unable to open the job_info file
   doing the analysis...
   WARNING: analysis failed...
./analysis_default_regression.py  PEC_particle_plt00020
ERROR: Benchmark and plotfile checksum have different value for key [electron,particle_position_z]
Benchmark: [electron,particle_position_z] 8.873588080118934e-36
Plotfile : [electron,particle_position_z] 9.312874618739578e-36
Absolute error: 4.39e-37
Relative error: 4.95e-02
ERROR: Benchmark and plotfile checksum have different value for key [lev=0,By]
Benchmark: [lev=0,By] 1.366852686440291e-16
Plotfile : [lev=0,By] 1.366868106441160e-16
Absolute error: 1.54e-21
Relative error: 1.13e-05
ERROR: Benchmark and plotfile checksum have different value for key [proton,particle_position_z]
Benchmark: [proton,particle_position_z] 2.149900391709847e-36
Plotfile : [proton,particle_position_z] 2.985886255355857e-36
Absolute error: 8.36e-37
Relative error: 3.89e-01

   execution time: 2.6s
   creating problem test report ...
   PEC_particle FAILED
```

**Comment by ax3l** [comment]
Looks safe to reset I think.

**Pull request merged**